tests/cli/test_cmds_start_stop_restart.py

```diff
@@ -132,6 +132,7 @@ def test_stop_or_start_handls_ls_remote_failures(
     mock_stdout,
 ):
     args = mock.Mock()
+    args.clusters = 'cluster1,cluster2'
```

Comment:
For the sake of avoiding a regression, it might be nice to keep a test that makes sure you can run this command with only one cluster specified (and maybe one making sure that no clusters defaults to all clusters).